test/api/controllers/RequestHandlerSpec.scala

```diff
@@ -121,72 +115,14 @@ class RequestHandlerSpec
 
     }
 
-    "given a request with a RequestCannotBeFulfilled gov-test-scenario header" when {
```

Comment:
Why is this removed? missing context for it

Reply:
Updated spec to include this